proc_scripts/align_slice.py

```diff
@@ -28,3 +28,27 @@ def align_and_slice(s, dimension='t2', convwidth=500, threshold=0.05, fl=None):
     if fl is not None:
         fl.pop_marker()
     return s[dimension:slices]
```

Comment:
(not pertaining to these particular lines)

note that there is a conflict below.  once these conversations are resolved, we can rebase this branch onto the most recent copy of master, which I think will make the conflicts go away

Reply:
okay I will wait for you to resolve then!

Reply:
I'm going to wait for your go-ahead.  I need to close this pull request (which will kill all the comments) before I can rebase.

Reply:
(i.e. is there anything in these comments that you would like addressed that is not currently addressed?)

Reply:
no, the only remaining issue is the enhancement curve spoof. I have fixed the reordering and so the only issue is the enhancement curve which i believe is due to it doing an enhancement curve for every run rather than the average.